TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts

```diff
@@ -18,7 +18,8 @@ export class TicketCenterComponent implements OnInit {
   ticketList: TicketStub[];
   pagination: {current: number, max: number } = {current: 1, max: 2};
   currentList;
-  listReady: Boolean = false;
+  sortingColumns: Set<String> = new Set();
```

Comment:
What's the advantage of using a set here?

Reply:
Set is so it only has unique values. We only care about the unique values for multisort. I could of done an array, and just checked before adding, but a set seemed easier.